server/tests/employer.form.test.js

```diff
@@ -214,142 +155,19 @@ describe('Server V1 Form Endpoints', () => {
-    expect(res).toEqual(
-      expect.arrayContaining(
-        [site].map((item) => (expect.objectContaining(item))),
-      ),
-    );
-  });
-
-  it('Gets a single site', async () => {
-    const res = await getSiteByID(1);
-    expect(res).toEqual(
-      expect.arrayContaining(
-        [site].map((item) => (expect.objectContaining(item))),
-      ),
-    );
-  });
-
-  it('gets a site before and after hires have been made', async () => {
-    const [res] = await getSiteByID(1);
-    expect(res.hcapHires).toEqual('0');
-    expect(res.nonHcapHires).toEqual('0');
-    await makeParticipant(participant1);
-    await makeParticipant(participant2);
-    const { data: [ppt, ppt2] } = await getParticipants({ isMoH: true });
-    await setParticipantStatus(employerAId, ppt.id, 'prospecting');
-    await setParticipantStatus(employerAId, ppt.id, 'interviewing');
-    await setParticipantStatus(employerAId, ppt.id, 'offer_made');
-    await setParticipantStatus(employerAId, ppt.id, 'hired', {
-      site: res.siteId,
-      nonHcapOpportunity: false,
-      positionTitle: 'title',
-      positionType: 'posType',
-      hiredDate: new Date(),
-      startDate: new Date(),
-    });
-    await setParticipantStatus(employerBId, ppt2.id, 'prospecting');
-    await setParticipantStatus(employerBId, ppt2.id, 'interviewing');
-    await setParticipantStatus(employerBId, ppt2.id, 'offer_made');
-    await setParticipantStatus(employerBId, ppt2.id, 'hired', {
-      site: res.siteId,
-      nonHcapOpportunity: true,
-      positionTitle: 'title',
-      positionType: 'posType',
-      hiredDate: new Date(),
-      startDate: new Date(),
-    });
-    const [res1] = await getSiteByID(1);
-    expect(res1.hcapHires).toEqual('1');
-    expect(res1.nonHcapHires).toEqual('1');
-  });
-
-  it('Create new site, receive validation error', async () => {
-    expect(
-      saveSites({ ...site, siteContactPhoneNumber: '1' }),
-    ).rejects.toEqual(new ValidationError('Phone number must be provided as 10 digits (index 0)'));
-  });
-
-  it('Create new site, receive duplicated', async () => {
-    const res = await saveSites(site);
-    const expectedRes = [
-      { siteId: 67, status: 'Duplicate' },
-    ];
-    expect(res).toEqual(expectedRes);
-  });
-
-  it('Update site, receive success', async () => {
-    const res = await updateSite(1, {
-      siteName: 'test',
-      history: [{ timestamp: new Date(), changes: [{ field: 'siteName', from: 'test1', to: 'test' }] }],
-    });
-    expect(res[0].siteName).toEqual('test');
-  });
-
-  it('fetches the list of employers', async () => {
+  it('fetches the list of employer expression of interest forms', async () => {
     const res = await getEmployers({
       isSuperUser: false, isMoH: false, isHA: true, regions: ['Vancouver Island'],
     });
     expect(res).toEqual([]);
   });
 
-  it('fetches a single employer', async () => {
+  it('fetches a single employer expression of interest form', async () => {
```

Comment:
Only "real" change in this file is updating the descriptions on lines 158 & 165 - the rest has just been moved to the sites test file